### DISCUSSION TITLE
Support for Frontend Submissions

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/publishpress-checklists#664] [Feature Requests]
for frontend post submission (Using JetEngine, JetFormBuilder) for a membership site it does not show any warnings and posts can be submitted but not published because of the restrictions.

My question is how can the meta be displayed to the users via Elementor template? Is there a shortcode or something that can be used for this?

https://secure.helpscout.net/conversation/2594140043/222516?viewId=6932143

**Comment by artfauxlive** [comment]
You should add to this that this is for the Publish Press Checklist Pro. It would be nice if a shortcode could be created to use in a template that would reveal the Checklist Meta that members could see before submitting their posts.
